src/js/gi-bill-comparison-tool/components/Dropdown.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+class Dropdown extends React.Component {
```

Comment:
All of our components should use the USWDS styles, so it doesn't look like it would change much, style-wise. But waiting to incorporate them until the rewrite is more settled seems reasonable to me.